### PR TITLE
Streamlit AgGrid for Species DataFrame

### LINKED ISSUE #9: Display Genus/Species, Time, and Location DataFrame
Display initial DataFrame list of genus/species, time, location for Genus and Species.

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #9

### CODE REVIEW

File: src/apps/spoc_verifier.py [hunk 1]
```diff
@@ -11,44 +12,41 @@
...
-    [PLACE]
+    if not selected_df.empty:
+        f"""
+        ## {selected_df['Species'][0]}
 
-    """
+        ## Places
+        """
+        for place in selected_df["Place"]:
+            place = eval(place)
```
Comment (jermnelson) [inline src/apps/spoc_verifier.py:46]: This `exec` call should be temporary, when users are able to edit the AgGrid, we'll want to sanitize `place`.

File: src/apps/spoc_verifier.py [hunk 2]
```diff
@@ -11,44 +12,41 @@
...
     """
     # SPOC Verifier
     """
-    st.dataframe(species)
+    records_grid = AgGrid(
+        species,
+        gridOptions=grid_options.build(),
+        data_return_mode=DataReturnMode.FILTERED,
+        update_mode=GridUpdateMode.MODEL_CHANGED,
+        allow_unsafe_jscode=True,
```
Comment (jgreben) [inline src/apps/spoc_verifier.py:32]: 😳Unsafe JS code?

**Review by jgreben — COMMENTED** [review]
Just wondering what that configuration means about unsafe JS code?

File: src/apps/spoc_verifier.py (hunk 2 above)
Comment (jermnelson) [reply src/apps/spoc_verifier.py]: Good catch! This is needed if we inject JS code (which we aren't doing now). Removing.

**Review by jgreben — APPROVED** [review]

File: src/apps/spoc_verifier.py (hunk 2 above)
Comment (jgreben) [reply src/apps/spoc_verifier.py]: I verified that it defaults to False.

**Pull request merged**